Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM ubuntu:jammy
+
+USER root
+ENV DEBIAN_FRONTEND=noninteractive
+RUN apt-get clean && rm -rf /var/lib/apt/lists/* && apt-get update -y
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    ca-certificates \
+    git \
+    git-lfs \
+    unzip \
+    wget \
+    zip \
+    adb \
+    openjdk-17-jdk-headless \
+    rsync \
+    && rm -rf /var/lib/apt/lists/*
+
+ARG GODOT_VERSION="4.3"
```

Comment:
This version is needed to not run into a crash when building, related to godot version 4.2.1 / PagedAllocator.h